triton_backend/all_models/llmapi/tensorrt_llm/1/model.py

```diff
@@ -268,6 +307,50 @@ def _response_loop(self):
                 if response_flag == pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL:
                     self._ongoing_request_count -= 1
 
+    def cancellation_loop(self):
+        """Checks if any pending requests have been cancelled."""
+        while self.running:
+            import time
+            time.sleep(.001)
```

Comment:
Let's make it a parameter right away in case this impact perf. Can we add it to the `triton_config` section of the .yaml?

Reply:
Sounds good, I added the setting. I didn't add it in the default yaml file, let me know if I should.